src/tests/ftest/recovery/pass_2.yaml

```diff
@@ -0,0 +1,23 @@
+hosts:
+  test_servers: 2
+
+timeout: 360
+
+server_config:
```

Comment:
I like how you put spaces between the sections. Maybe we should try to standardize this

Reply:
I was doing this for a while because it's easier to read, but nobody else is doing it, so I thought I wasn't following the standard.

Reply:
Our "standards" tend to be unspoken - something we really need to document at some point... But I personally prefer the spaces, and @phender I think does as well. Not sure about others